Tasks/DownloadPipelineArtifactV0/task.json

```diff
@@ -43,7 +43,7 @@
     "instanceNameFormat": "Download Pipeline Artifact",
     "execution": {
         "AgentPlugin": {
-            "target": "Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTask, Agent.Plugins"
+            "target": "Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTaskV0, Agent.Plugins"
```

Comment:
you need min agent version for the V0 task as well, since the latest version task can't run on old version agent.
old agent only understand `Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTask`

Reply:
Will change thanks!